### PR TITLE
Feature/second milestone

### PR DESCRIPTION
**Pull request by Jmagero**
### Milestone 2 User_interface

- [X] Created user command interface
- [X] Added input from the user

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+    player_current = player_one
+    available_moves = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+    # Players turn instruction
+    puts "#{player_current} It is your turn."
+    print_board(available_moves)
+    puts "Please input your move. You have these moves available: #{available_moves}"
+    current_player_selected_move = gets.chomp
+    available_moves[current_player_selected_move.to_i - 1] = 'X'
+    game_invalid_move = true
```
Comment (mosesogwo) [inline bin/main.rb]: This should be a conditional
game_invalid_move is only true if the preceding line is true

Comment (mosesogwo) [reply bin/main.rb]: Also, have it in mind that moves outside the range of 1-9 are also invalid.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @Jmagero, @santiagorodriguezbermudez :clap:. Impressive work :+1:**

I can see that you have thought of these actions:
:heavy_check_mark: game asks players for their names
:heavy_check_mark: game tells which player turn it is
:heavy_check_mark: game asks the player to select from available moves
:heavy_check_mark: game informs player if the selected move is invalid
:heavy_check_mark: game displays board after the player move

Here are things missing in your consideration:
:x: game repeats all actions for the next player's move - game turn should be repeated for the second player
:x: game_not_finished status at the end of each player turn should be determined by whether there is a win/draw. Win or draw should be checked before game_not_finished status is changed.
:point_right: Modify your code to reflect the above points.


_**Kindly make these changes and submit another code review request.**_

**Any questions? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Comment by santiagorodriguezbermudez** [comment]
# Milestone 2 User_interface - Code Review # 1
## We fix the following on this code review

- [X] Added Pseudo Logic for changing players: game repeats all actions for the next player's move - game turn should be repeated for the second player
- [X]  Added comments to show where this will be address in the next milestone. game_not_finished status at the end of each player turn should be determined by whether there is a win/draw. Win or draw should be checked before game_not_finished status is changed.

## Warning: The game logic has not been implemented. Therefore, the winner and changing the current player will be solved on milestone 3

**Review by mosesogwo — APPROVED** [review]
**Great! Milestone is approved :heavy_check_mark:**

**Good luck in your next milestone :stars:**

**Pull request merged**